stencil-library/src/components/form/form-control-number.scss

```diff
@@ -17,3 +17,17 @@ input[type='number']::-webkit-outer-spin-button {
 input[type='number'] {
   -moz-appearance: textfield;
 }
+
+input[type='number'][disabled] {
```

Comment:
Instead of creating styles, can we use the bootstrap default styles that get applied to `disabled` form controls? https://getbootstrap.com/docs/5.3/forms/select/#disabled

It looks like you can also use the `disabled` attribute on a `fieldset` to disable the whole fieldset as shown here: https://getbootstrap.com/docs/5.3/forms/overview/#disabled-forms

Reply:
Nice!